sql/core/src/main/scala/org/apache/spark/sql/execution/Sort.scala

```diff
@@ -93,4 +98,63 @@ case class Sort(
       sortedIterator
     }
   }
+
+  override def upstream(): RDD[InternalRow] = {
+    child.asInstanceOf[CodegenSupport].upstream()
+  }
+
+  // Name of sorter variable used in codegen.
+  private var sorterVariable: String = _
```

Comment:
Why? This is the state that needs to be kept between the two member functions in this class.